internal/services/sources_service.go

```diff
@@ -10,21 +10,26 @@ import (
 	"github.com/go-chi/render"
 )
 
+// var sourceTypeName = map[string]int{"google": 7, "amazon": 5, "azure": 12}
+
```

Comment:
You will find find our cloud constants at model.ProviderType (that's the type, there are three constants there).

Are these numbers 7, 5, 12 from sources database? What are these? Database IDs?

Reply:
I am afraid we will have to perform an extra query to fetch SourceType and from there we get the proper database ID.

Reply:
Oh it looks like sources support some kind of advanced filtering, that could actually work (filter by source type name): https://github.com/RedHatInsights/sources-api-go/blob/2d5f69436137b7e8e3f96fccb35c50355f5fe140/util/filtering.go#L19

Need to dig deeper.

Reply:
yes, these are the ids from sources regarding the source_type_id when I looked at /source_types

Reply:
This looks, better, now, we must not expose other's constants/names through our API. We need our own ids/names for target environments. Can you write code which will actually map the three providers we have defined in `model_constants.go` to SourceType names?

I think we should stick with our own naming convention, whatever we end up using we should use for all our API endpoints. I named the constants as "providers", but let's pick better term if folks don't like it.